src/doc_builder/commands/build.py

```diff
@@ -176,8 +176,8 @@ def build_command_parser(subparsers=None):
     parser.add_argument(
         "--version",
         type=str,
-        help="Version under which to push the files. Will not affect the actual files generated, as these are"
-        " generated according to the `path_to_docs` argument.",
+        help="Version of the documentation to generate. Will default to the version of the package module (using "
+        "`master` for a version containing dev.",
```

Comment:
```suggestion
        "`master` for a version containing dev).",
```

What does the package module refer to and "version containing dev". How does it get the default version (read the git tag, ...?)

Reply:
The module whose doc is being built. It has a `__version__` attribute containing its version.

As for version containing dev, it's the standard way of representing a version in development in Python, see for instance the [current version](https://github.com/huggingface/transformers/blob/7f921bcf47f2d6875feb6c1232dc6cfbeb4668e8/setup.py#L356) in Transformers.